src/utils/netCoreUtils.ts

```diff
@@ -17,7 +18,13 @@ export async function getNetCoreProjectInfo(target: 'GetBlazorManifestLocations'
     const command = `dotnet build /r:false /t:${target} /p:CustomAfterMicrosoftCommonTargets="${targetsFile}" /p:CustomAfterMicrosoftCommonCrossTargetingTargets="${targetsFile}" /p:InfoOutputPath="${outputFile}" "${project}"`;
 
     try {
-        await execAsync(command, { timeout: 10000 });
+        try {
+            await execAsync(command, { timeout: 20000 });
+        } catch (err) {
+            const error = parseError(err);
+            throw new Error(localize('vscode-docker.netCoreUtils.noProjectInfo', 'Unable to determine project information for target \'{0}\' on project \'{1}\' {2}', target, project, error.message));

```

Comment:
I am not entirely sure if `localize()` requires the message key and text to be spelled out verbatim (so tools can fish them out of code and generate localization files automatically), or if they can be defined as constants somewhere... 

Assumed the former, hence the duplication on line 25 and 37.

Reply:
Generally I would just do two separate strings (even if identical), two separate keys here. It might change with the `l10n` API though. Ultimately I don't think it really matters though, unless we ever _actually_ do localization.